HW3/Training-HW3-flex/index.html

```diff
@@ -5,51 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Training HW3</title>
-    <link rel="stylesheet" href="css/styles.css" />
     <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
     <div class="container">
```

Comment:
The `container` shouldn't be applied for the whole page

![image](https://user-images.githubusercontent.com/44003977/218419813-3da85679-016e-4072-818c-77fcce85d21f.png)